pygfx/renderers/wgpu/_renderer.py

```diff
@@ -692,3 +706,61 @@ def snapshot(self):
         )
 
         return np.frombuffer(data, np.uint8).reshape(size[1], size[0], 4)
+
+    def enable_events(self):
+        # TODO: check not a texture
```

Comment:
don't forget to add this check :)

Reply:
I've added a check for `hasattr`. Should be enough, right?